### DISCUSSION TITLE
There does not seem to be any documentation around responsive layout

### DISCUSSION THREAD

**Discussion by akmittal** [twilio-labs/paste#1954] [General] [answered]
Most of component accept responsive style array values. But this feature is not documented anywhere about what all beakpoints are there for a particular theme. Is these any plan to include same?

**Comment by kate-porter** [comment, marked as answer]
Hey @akmittal - while we will likely create responsive guidelines _at some point_, there's no set plan or timing at this point. A large majority of Twilio's products right now do not support responsive layouts, so it's not been bubbled up very high on the backlog. Do you have a need for responsive layouts in your work?

@twilio-labs/design-systems-pd will be creating Spacing and Layout guidelines quite soon (within the next month or so), so we could consider including some guidelines around responsive layouts when we do that. I'd be interested to hear more about what type of documentation might be helpful for you in your work.

**Reply by akmittal** [reply]
Hi @katieporter 
We recently launched https://showcase.twilio.com and made it responsive using paste. Problem was there was no documentation for what all breakpoints are set. So I think it will helpful if breakpoints for all the themes are documents for time being.

**Reply by SiTaggart** [reply]
We don't actually have any opinions on breakpoints. All components are 100% width of their container so they will work with any set of responsive breakpoints.

On the theme provider you can provide Paste you're own set of breakpoints https://paste.twilio.design/tokens/theme-package#props.

The responsive array props will respect the number of breakpoints you provide to Paste. So, if you set 5 custom breakpoints, you can provide a 5 item array responsive values for any responsive style prop.

The docs need refining a lot, but most folks aren't doing responsive design so we haven't prioritized it. 

Hopefully that works for your needs

**Reply by holylander** [reply]
I believe the default Twilio Paste break points are described here https://paste.twilio.design/foundations/spacing-and-layout#max-widths
`25rem (400px), 64rem (1024px), and 77rem (1232px)`
